compiler/testData/codegen/box/annotations/parameters.kt

```diff
@@ -1,3 +1,5 @@
+// DONT_TARGET_EXACT_BACKEND: WASM
+// WASM_MUTE_REASON: IGNORED_IN_JS
```

Comment:
How WASM_MUTE_REASON is used?

Reply:
It is just a comment. Consistent format makes it better for searching.